docs/UserGuide.md

```diff
@@ -14,9 +14,9 @@ ProfNUS is a **desktop application which helps SOC Professors who have many modu
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `ProfNUS.jar` from [here](https://github.com/AY2223S1-CS2103T-W11-2/tp/releases).
+1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
```

Comment:
I think you might not have pulled from master before doing the changes for the UserGuide.